functions/go/fmt/README.md

```diff
@@ -0,0 +1,73 @@
+# fmt
+
+### Overview
+
+<!--mdtogo:Short-->
+
+Format resources
+
+<!--mdtogo-->
+
+### Synopsis
+
+<!--mdtogo:Long-->
+
+Format formats the field ordering in YAML configuration files. Field ordering
+roughly follows the ordering defined in the source Kubernetes resource definitions
```

Comment:
can you expand on this? Where does Go struct come into play for arbitrary CRDs?

Reply:
Removed Go struct part. Arbitrary CRs are formatted lexically.